lightning/src/ln/channelmanager.rs

```diff
@@ -1299,6 +1299,13 @@ pub(super) struct PeerState<SP: Deref> where SP::Target: SignerProvider {
 	/// will remove a preimage that needs to be durably in an upstream channel first), we put an
 	/// entry here to note that the channel with the key's ID is blocked on a set of actions.
 	actions_blocking_raa_monitor_updates: BTreeMap<ChannelId, Vec<RAAMonitorUpdateBlockingAction>>,
+	/// The latest [`ChannelMonitor::get_latest_update_id`] value for all closed channels as they
+	/// exist on-disk/in our [`chain::Watch`]. This *ignores* all pending updates not yet applied
```

Comment:
s/ignores/excludes ?

Reply:
Either is fine, I suppose. "ignores" seems more like an action whereas "excludes" seems more to described a state.